server/core/scheduler/model/src/main/java/com/antgroup/openspg/server/core/scheduler/model/service/SchedulerInstance.java

```diff
@@ -0,0 +1,269 @@
+/*
+ * Copyright 2023 Ant Group CO., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied.
+ */
+
+/** Alipay.com Inc. Copyright (c) 2004-2023 All Rights Reserved. */
+package com.antgroup.openspg.server.core.scheduler.model.service;
+
+import com.antgroup.openspg.server.common.model.base.BaseModel;
+import java.util.Date;
+
+/**
+ * @author yangjin
+ * @version : SchedulerInstance.java, v 0.1 2023年11月30日 09:50 yangjin Exp $
```

Comment:
please write comments in English as clearly as possible, and there should be no internal personal account information.